nodejs/rest-api/tests/rest.api.test.js

```diff
@@ -0,0 +1,35 @@
+console.clear();
+
+require('dotenv').config({path: '.env'});
+
+const { expect } = require('chai');
+const restApiHandler = require('../handlers/rest.api.handler');
+
+describe('Rest API tests', function() {
+
+    it('should be able to get the account by id', async function() {
+        const contracts = await restApiHandler.getAccountById(process.env.OPERATOR_ID)
+        expect(contracts).to.have.property('status').equals(200);
+    });
+
+    it('should be able to get the contracts', async function() {
+      const contracts = await restApiHandler.getContracts();
+      expect(contracts).to.have.property('status').equals(200);
```

Comment:
More detail check the payload here as well (see what i do for eth.logs under the json-rpc relay if u need better examples